iree/base/logging.cc

```diff
@@ -91,6 +105,13 @@ void LogMessage::EmitLogMessage() {
   fprintf(stderr, "%c %s:%d] %s\n", "IWEF"[severity_], file_name_, line_,
           str().c_str());
 
+#if defined(__ANDROID__)
+  auto android_severity = kStatusToAndroidLevel[severity_];
+  auto android_message =
```

Comment:
nits on use of `auto`:
  * `int` is shorter, why not use it? 
  * `std::string` is used explicitly below with `absl::StrFormat`, could be consistent here

```suggestion
  int android_severity = kStatusToAndroidLevel[severity_];
  std::string android_message =
```